src/main/java/org/spongepowered/api/attributes/Attributes.java

```diff
@@ -0,0 +1,15 @@
+package org.spongepowered.api.attributes;
+
+public class Attributes {
```

Comment:
Might want to denote the class as `final` as well. Assuming this is a quasi enumeration class like `DyeColors` and such.